docs/pages/deploy-a-cluster/helm-deployments/custom.mdx

```diff
@@ -94,7 +94,7 @@ proxy:
         severity: INFO
     proxy_service:
       enabled: true
-      listen_addr: 0.0.0.0:3080
+      listen_addr: 0.0.0.0:443
```

Comment:
I'd check with @hugoShaka, but this might be a breaking change if the helm deployment uses a kube service on port 443 and the proxy service still listens on port 3080 on the backend.

Reply:
Nice catch :) The chart expects your proxy to listen on 3080 as the Kubernetes Service resource is already doing the 443 -> 3080 translation. 

Listening on port 3080 in a container is easier and better for security because creating listeners for ports < 1024 requires higher privileges. We're currently trying to harden the containers and chart and running on privileged ports might not be possible in a near future.

For those reasons, I strongly advise keeping the 3080 listener in kube-hosted configuration and maybe adding a comment explaining Kubernetes is redirecting 443 on the service to 3080 on the pod.